e2e-tests/tests/monitoring/01-deploy-pmm-server.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      deploy_pmm_server
+
+      api_key=$(generate_pmm_api_key)
```

Comment:
Like my other comment about renaming `generate_pmm_api_key` maybe rename this to `server_key`. Applies to other places.